webpack.config.js

```diff
@@ -6,6 +6,7 @@ var Router = require('react-router');
 var routes = require('./components/routes.jsx');
 var paths = require('./scripts/paths.js');
 var currencies = require('./scripts/currencies.js');
+var englishStrings = require(path.join(__dirname, 'locales/en-US.json'));
```

Comment:
What's the purpose of defining this way up here? Or was it just cleaning things up/as good as anything?